deploy/olm-catalog/kogito-cloud-operator/0.6.0/kogito-cloud-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -60,87 +60,87 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
```

Comment:
There's a lot of changes in this file. Have you upgraded to 0.11 SDK?

Reply:
updated.